docs/indicators/url-extraction.md

```diff
@@ -0,0 +1,104 @@
+---
+id: url-extraction
+title: URL Extraction
+---
+
+
+The Cortex XSOAR URL indicator type is built using regex and a formatting script.
+The following describes the URL indicator components and what output you should expect when extracting indicators of type URL.
+
+## URL Indicator Components

```

Comment:
Its the extraction components :) the URL indicator has more components such as reputation and enrichment

Reply:
The URL itself is built using regex and a formatting script.
The reputation and enrichments are post-creation.

Reply:
true :) but they are still part of the URL component

Reply:
So I think it shouldn't be called 'URL Extraction' but 'URL Indicator' with the all details.